### DISCUSSION TITLE
question about python

### DISCUSSION THREAD

**Discussion by alexandris-cyber** [community/community#21513] [Other Feature Feedback, Questions, & Ideas]
<p>I have no backround in computer programming.Why I cant add these numbers?it says invalid character in identifier.</p>
<p>thank you </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f398980281fde60817326bfd4a2e2c2c99827dfd.jpeg" data-download-href="/uploads/short-url/yKWXgScYFtplVHZ2jSEZqKznGrj.jpg?dl=1" title="question about python.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/61056385/181065074-5b214dcd-171d-4b6c-86fe-67b163d718e0.jpg" alt="question about python.jpg" data-base62-sha1="yKWXgScYFtplVHZ2jSEZqKznGrj" width="690" height="367" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/f398980281fde60817326bfd4a2e2c2c99827dfd_2_690x367.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/f/f398980281fde60817326bfd4a2e2c2c99827dfd_2_1035x550.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/f/f398980281fde60817326bfd4a2e2c2c99827dfd.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f398980281fde60817326bfd4a2e2c2c99827dfd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">question about python.jpg</span><span class="informations">1365×727 104 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! The error is that you are using “===”. If you delete those equal signs on lines number 2 and 5 you should be okay. </p>
<p>Also, note that the words “Spyder Editor” will give you an error unless you put quotation marks around them. </p>

**Comment by alexandris-cyber** [comment]
<p>thank you veryyyyyy much.Now is it ok? I deleted &gt;&gt; because it was said invalid syntax again <img src="https://user-images.githubusercontent.com/61056385/181065077-4b68f2a7-b477-4623-bcfc-eaa753145881.jpeg" alt="add problem 1.jpg" data-base62-sha1="k68Yy217PDUnviLVGU00k81g1SC" width="624" height="205"> <img src="https://user-images.githubusercontent.com/61056385/181065078-77d9628d-949a-4f86-97ba-0e3035ecd91b.jpeg" alt="add problem 2.jpg" data-base62-sha1="4sqX9MOzj11j5irB7rjZhHAO5Xm" width="644" height="232"> </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/597bee8c87175bd7231b37f9eb2209ca4fcf1073.jpeg" data-download-href="/uploads/short-url/cLC15DnmNr5D6cEnYBjBObyLDeH.jpg?dl=1" title="add problem 3.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/61056385/181065079-6a681c4b-d887-4096-94d0-570c256b6b58.jpg" alt="add problem 3.jpg" data-base62-sha1="cLC15DnmNr5D6cEnYBjBObyLDeH" width="690" height="369" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/597bee8c87175bd7231b37f9eb2209ca4fcf1073_2_690x369.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/5/597bee8c87175bd7231b37f9eb2209ca4fcf1073_2_1035x553.jpg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/597bee8c87175bd7231b37f9eb2209ca4fcf1073.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/597bee8c87175bd7231b37f9eb2209ca4fcf1073_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">add problem 3.jpg</span><span class="informations">1365×730 99.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by logankilpatrick** [comment]
<p>Yes, also keep in mind that if you want to see the output of something (like “2+3”) in the console, you need to print it like “print(2+3)”. If you add that you will see “5” show up on the right side of the screen in the console. </p>

**Comment by alexandris-cyber** [comment]
<p>i am very grateful to you!!!you make me feel like ease into building the base in how to programm in python.</p>
